src/sidebar/store/modules/activity.js

```diff
@@ -39,6 +43,33 @@ const update = {
     };
   },
 
+  ANNOTATION_SAVE_STARTED(state, action) {
+    let addToStarted = null;
+    if (
+      action.annotation.$tag &&
+      !state.activeAnnotationSaveRequests.includes(action.annotation.$tag)
+    ) {
+      addToStarted = action.annotation.$tag;
+    }
+    const updatedSaves = state.activeAnnotationSaveRequests.concat(
```

Comment:
Could this be a problem when the condition above fails? It looks like its going to add `null` to the array because addToStarted defaults as `null`

Reply:
Yeeeesss, this is definitely bad. I guess I'm moving a little fast today. I'll make this initialize to an empty Array and then push a `$tag` to it if valid.

Reply:
This has been resolved.